File: ios/PrinterStarIos.m
#import <React/RCTBridgeModule.h>

// bridging with swift
@interface RCT_EXTERN_MODULE(PrinterStarIos, NSObject)\
    RCT_EXTERN_METHOD(callbackMethod:(RCTResponseSenderBlock)callback)
    RCT_EXTERN_METHOD(simpleMethod:(NSString *)message)
@end

File: ios/PrinterStarIos.swift
//
//  PrinterStarIos.swift
//  PrinterStarIos
//
//  Created by Imron Imam Wahyudi Gunawan on 27/01/20.
//  Copyright © 2020 Facebook. All rights reserved.
//

import Foundation

@objc(PrinterStarIos)
class PrinterStarIos: NSObject {
    
    @objc(callbackMethod:)
    func callbackMethod(callback: RCTResponseSenderBlock) -> Void {
      let resultsDict = [
        "success" : true
      ];
       
      callback([NSNull() ,resultsDict])
    }
    
    @objc(simpleMethod:)
    func simpleMethod(message: String) {
      print("\(message)")
    }
    
}
